### ISSUE TITLE
Set up donate button and other short-term finance/accounting systems/software/infrastructure

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/iced-coffee#340]
**Task description:** 
- We need to setup our own donate button.
- Start convo in #finance, about short-term and medium term accounting/fundraising software options
- Reinvestigate eligibility to use [Open Collective](https://opencollective.com/)

**Owner:** Person who completes the task. Also the "Assignee"
@mateoclarke  

**Supporter:** People available to help with the task
@JMensch 

**Next Steps:** 
- [x] Start a convo in #finance
- [x] Research & Implement Donation systems to replace what we had via CfA
- [ ] ~Create a plan for 501c3 Working Group to act on creating non-profit roadmaps~

**Comment by mateoclarke** [comment]
Posting notes as an update from @JMensch and I getting together to try to get more organized on next steps: https://docs.google.com/document/d/1rhit58w16fn7TJGtTyc54qt-9AVEkyRLBMdrWNgNlbo/edit#

**Comment by JMensch** [comment]
Blocked by #355

**Comment by JMensch** [comment]
PR for new donate button

**Comment by mateoclarke** [comment]
Moving this item to a new issue:
"Create a plan for 501c3 Working Group to act on creating non-profit roadmaps"
https://github.com/open-austin/iced-coffee/issues/349#issuecomment-478813580
